sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -44,7 +50,11 @@ import org.apache.spark.sql.catalyst.plans.logical.{LogicalPlan, Project}
  *                 unless they are specified with full qualified table name with database prefix.
  */
 case class CreateViewCommand(
-    tableDesc: CatalogTable,
+    name: TableIdentifier,
+    userSpecifiedColumns: Seq[(String, Option[String])],
+    comment: Option[String],
+    properties: Map[String, String],
+    originalText: Option[String],
     child: LogicalPlan,
     allowExisting: Boolean,
     replace: Boolean,
```

Comment:
btw, we can rename `allowExisting` to `ignoreIfExists`. I am not sure we need to `SaveMode` (a public API) to catalyst.